### PR TITLE
Add Prisma Fastify example

### PR DESCRIPTION
**Pull request by keturiosakys**
This PR adds a basic example of an instrumented Fastify API using Prisma (sqlite) as database ORM.

### CODE REVIEW

**Review by emschwartz — COMMENTED** [review]
I think if you're going to show an example using a DB client, it would be worth having a separate DB method that you can use to illustrate instrumenting an internal function with autometrics

File: .gitignore [hunk 1]
```diff
@@ -46,3 +47,5 @@ lerna-debug.log*
 !.vscode/tasks.json
 !.vscode/launch.json
 !.vscode/extensions.json
+
+
```
Comment (oscarvz) [inline .gitignore:51]: Unnecessary additional newline.
```suggestion
```

File: examples/fastify/.env.example [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="file:./dev.db"
```
Comment (oscarvz) [inline examples/fastify/.env.example:7]:
```suggestion
DATABASE_URL="file:./dev.db"

```

File: examples/fastify/src/index.ts [hunk 3]
```diff
@@ -1,34 +1,57 @@
...
+}
+
+// Example internal function calling a database instrumented with autometrics
+const createTulip = autometrics(async function createTulip(
+  tulip: Tulip,
+) {
+  return prisma.tulip.create({
+    data: tulip,
+  });
+});
```
Comment (oscarvz) [inline examples/fastify/src/index.ts:33]: I'd prefer the `createTulip` to be declared separately, which then can be wrapped. The intention is much clearer this way, and makes the comment redundant.

```ts
async function createTulip(tulip: Tulip) {
  return prisma.tulip.create({
    data: tulip,
  });
}

const createTulipAutometricsInstrumented = autometrics(createTulip);
```

**Review by oscarvz — APPROVED** [review]
Apart from a few comments: looking good! 🚀

**Pull request merged**